<launch>
  <!-- to run gdb on the node, provide "doDebug:=true" /-->
  <arg name="doDebug" default="false" />
  <arg if="$(arg doDebug)" name="launchPrefix" value="xterm -e gdb -args" />
  <arg unless="$(arg doDebug)" name="launchPrefix" value="" />

  <arg name="debug_line_detector" default="false" />
  <arg name="save_images" default="false" />
  <arg name="show_image" default="false" />

  <!-- Start relay node: -->
  <node name="line_detector_node" pkg="line_detector" type="line_detector_node" output="screen" launch-prefix="$(arg launchPrefix)" >
    <param name="debug" value="$(arg debug_line_detector)" type="bool" />
    <param name="save_images" value="$(arg save_images)" type="bool" />
    <param name="show_image" value="$(arg show_image)" type="bool" />
  </node>

</launch>